build.sbt

```diff
@@ -1,9 +1,43 @@
+import scala.sys.process.Process
+import java.util.Base64
+
 lazy val library = (project in file("."))
   .disablePlugins(JUnitXmlReportPlugin) // Required to prevent https://github.com/scalatest/scalatest/issues/1427
   .settings(
     name := "ui-test-runner",
     majorVersion := 0,
     scalaVersion := "2.13.13",
     crossScalaVersions := Seq("2.13.13", "3.3.3"),
-    libraryDependencies ++= Dependencies.compile
+    libraryDependencies ++= Dependencies.compile,
+    Compile / resourceGenerators += Def.task {
+      val encodedChromeExtension =
+        (Compile / resourceManaged).value / "extensions" / "chrome" / "accessibility-assessment"
+
+      val chromeExtensionSource =
+        baseDirectory.value / "browser-extensions" / "accessibility-assessment"
+
+      Process("npm install", chromeExtensionSource).!
+      Process("npm run build", chromeExtensionSource).!
+
+      val chromeExtensionDist = chromeExtensionSource / "dist"
+
+      val chromeExtensionFiles = IO.listFiles(chromeExtensionDist).toList
+
+      val zipFileMappings = chromeExtensionFiles.map { file =>
+        (file, chromeExtensionDist.toURI.relativize(file.toURI).getPath)
+      }
+
+      IO.withTemporaryDirectory { tmpDir =>
+        val chromeExtensionZip = tmpDir / "dist.zip"
```

Comment:
I guess you could name it as `accessibility-assessment-extension.crx` too

and you could have a tax to generate the crx file somewhere else so that people can grab it / upload it to the github releases manually